src/streams/send.rs

```diff
@@ -470,6 +475,14 @@ impl StreamTx {
         let seq_no = next.seq_no;
         self.last_used = now;
 
+        // This is set here due to borrow checker.
+        if self.pt_for_rtx != set_pt_for_rtx {
```

Comment:
Can we have a comment here that explains why we do this?

Reply:
Grrr. Writing the code doc, I realise this solution is not good enough either.

If the user switches PT mid session, the resends already queued must be sent with the resend-PT they belong to.

Reply:
Well. Sort of. I'll make it so we don't enqueue any resend requests for a PT that doesn't have an RTX PT.